File: olo-promotions-sdk-go/pkg/promotions/models/requests/accruals/void_accrual_request.go
package accruals

// VoidAccrualRequest represents a request to void a loyalty accrual.
type VoidAccrualRequest struct {
	// OrderID is the ID of the order, which is only populated once the order has been placed.
	OrderID string `json:"orderId"`

	// AccountID is the loyalty account's unique identifier in the provider's system.
	AccountID string `json:"accountId"`

	// Brand is a unique ID representing the restaurant brand in Olo's system.
	Brand string `json:"brand"`

	// StoreNumber is the ID of the store as provided by the restaurant.
	StoreNumber string `json:"storeNumber"`

	// Restaurant is a unique ID representing the vendor restaurant in Olo's system.
	Restaurant string `json:"restaurant"`
}

File: olo-promotions-sdk-csharp/OloLabs.Promotions.SDK/Requests/VoidAccrualRequest.cs
namespace OloLabs.Promotions.SDK.Requests
{
    public class VoidAccrualRequest
    {
        /// <summary>
        /// The ID of the order, which is only populated once the order has been placed.
        /// </summary>
        public string OrderId { get; set; }
        /// <summary>
        /// The loyalty account's unique identifier in the provider's system.
        /// </summary>
        public string AccountId { get; set; }
        /// <summary>
        /// A unique ID representing the restaurant brand in Olo's system.
        /// </summary>
        public string Brand { get; set; }
        /// <summary>
        /// The ID of the store as provided by the restaurant.
        /// </summary>
        public string StoreNumber { get; set; }
        /// <summary>
        /// A unique ID representing the vendor restaurant in Olo's system.
        /// </summary>
        public string Restaurant { get; set; }
    }
}